WooCommerce/Classes/ViewModels/Order Details/Shipping Labels/ShippingLabelPrintingInstructionsView.swift

```diff
@@ -0,0 +1,51 @@
+import SwiftUI
+
+/// Displays an illustration at the top and four steps on how to print a shipping label on an iOS device.
+struct ShippingLabelPrintingInstructionsView: View {
+    var body: some View {
+        ScrollView {
+            VStack {
+                Spacer()
+                    .frame(height: 20)
+                Image("woo-shipping-label-printing-instructions")
+                Spacer()
+                    .frame(height: 30)
+                ShippingLabelPrintingStepView(viewModel: .init(stepIndex: 1, text: Localization.step1, isLastStep: false))
```

Comment:
I understand using .init here makes the code very compact, but, to me, unless you check the code of ShippingLabelPrintingStepView to see what type of parameter it accepts, it makes difficult to understand what is happening, and what type those view models represent. What do you think?

Reply:
hmm interesting point, the main reason I used `.init` here is because the parameter name already suggests a "view model" so it's most likely a view model for `ShippingLabelPrintingStepView` (and usually called `*.ViewModel` or `*ViewModel`). in code, we can click into the `.init` to see the view model as well and it could save a few lines of code if the view model name is long (and save time for reformatting if we ever refactor the name of the view model). I generally find it easier to read with `.init` rather than the full class/struct name to focus on the parameters for the view model, but I could see the point of not knowing the type.

@woocommerce/ios-developers any thoughts on this?